docs/0.4/tutorials/using_the_admin_ui.md

```diff
@@ -0,0 +1,389 @@
+
+1. To make Alice's key active, follow these steps.
+
+   a. Hover over the key, then click **Set active**.
+
+   ![](../images/adminui_profile_alice_set_active_key.png
+   "Admin UI: Set active key")
+
+   b. Enter Alice's password when prompted.
+
+   c. If the operation was successful, the key is now marked as **Active**.
+
+   ![](../images/adminui_profile_alice_active_key.png
+   "Admin UI: Profile page with active key")
+
+> NOTE: To repeat this procedure for Bob, use a separate browser window that
+> points to `localhost:3031`.
+> In the `generate-registry` container, Bob's keys are available in the files
+> `/registry/bob.pub` and `/registry/bob.priv`.
+
+
+## Proposing a Circuit
+
+1. As Alice, click on the Circuits icon in the left menu. The Circuits page
+   displays a summary of existing circuits and proposals. In this example,
+   nothing exists yet.
+
+    ![](../images/adminui_circuits_page_blank.png
+    "Admin UI: Circuits summary page")
+
+
+1. Click **Propose New Circuit**. The **Add nodes** form shows the nodes that
+  are available for the circuit, with the local node (`acme-node-000`)
+  preselected. Select the checkbox for **Node beta-node-000**, then click
+  **Next**.
+
+    ![](../images/adminui_propose_2_add_beta.png
+    "Propose Circuit: Add beta node")
+
+1. On the **Add services** form, enter the following service information for
+   the alpha node.
+
+   | Item | Value |
+   |------|-------|
+   | Service type | `scabbard` |
+   | Service ID | `abdc` |
+   | Allowed node | `alpha-node-000` |
+   | Service argument | &nbsp; Key: `admin_keys` |
+   |                  | &nbsp; Value: `["<Alice's public key>"]` |
+   | Service argument | &nbsp; Key: `peer_services` |
+   |                  | &nbsp; Value: `["fghi"]` |
+
+   <br>
+
+   Tip: To add a second service argument, click the small **+** icon on the
+   right.
+
+   ![](../images/adminui_propose_3_add_services_alpha.png
+   "Propose Circuit: Add alpha service")
+
+4. Click the green check to submit alpha's service information. If the operation
+   was successful, the green check changes to a pencil (edit) icon.
+
+   ![](../images/adminui_propose_4_submit_service_alpha.png
+   "Propose Circuit: Alpha service confirmed")
+
+   Otherwise, an error displays so you can correct the information and
+   resubmit it.
+
+5. Click the pink **+** icon at the bottom of the form to add another service
+   form, then enter the service information for the beta node.
+
+   | Item | Value |
+   |------|-------|
+   | Service type | `scabbard` |
+   | Service ID | `fghi` |
+   | Allowed node | `beta-node-000` |
+   | Service argument | &nbsp; Key: `admin_keys` |
+   |                  | &nbsp; Value: `["<Bob's public key>"]` |
+   | Service argument | &nbsp; Key: `peer_services`
+   |                  | &nbsp; Value: `["abcd"]` |
+
+   <br>
+
+   Note: If you have not added Bob's keys and marked them as active, you can
```

Comment:
This note should be removed, the admin keys need to be set to the same key (or keys) always

Reply:
Changed the note to confirm that you must use the same key here, because this is unexpected (the other service values reflect the perspective of the beta node).